<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Disk size</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/scolasync.svg</normaloff>images/scolasync.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QToolButton" name="toolButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/icons16/devices/multimedia-player.png</normaloff>images/icons16/devices/multimedia-player.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <widget class="QLabel" name="label_total">
      <property name="text">
       <string>total size</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_used">
      <property name="text">
       <string>Used</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>363</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
